src/MBC_UserImport_Source_AfterSchool.php

```diff
@@ -194,6 +197,10 @@ public function addWelcomeSMSSettings($user, &$payload) {
       if (isset($user['hs_id'])) {
         $payload['hs_id'] = $user['hs_id'];
       }
+      if (isset($user['optin'])) {
```

Comment:
Not sure, but shouldn't this be `$user['afterschool_optin']`?
You're setting this to `$this->importUser['afterschool_optin']` in `setter()` above, and then [passing](https://github.com/DoSomething/mbc-user-import/blob/Issue53-afterschool_optin/src/MBC_UserImport_Source_AfterSchool.php#L115) `$this->importUser` as an argument to `addWelcomeSMSSettings()`.